### PR TITLE
bip 2: allow markdown

### PR DESCRIPTION
**Pull request by achow101**
BIP authors should be allowed to write their BIPs in Markdown. The MediaWiki syntax that GitHub uses appears to be entirely undocumented (it does not always match mediawiki's documentation) which makes writing complex BIPs rather difficult. Markdown is much simpler and more concise.

### CODE REVIEW

**Comment by kallewoof** [comment]
I think this is an acceptable amendment. @luke-jr

**Comment by Roasbeef** [comment]
Omg, this would be amazing! 

Mediawiki just feels so cumbersome compared to markdown.

**Comment by spedroza11** [comment]
My problem is not fixed

**Review by guggero — APPROVED** [review]

**Review by katesalazar — CHANGES_REQUESTED** [review]
Gonna guess this is missing changing `buildtable` around this line: https://github.com/bitcoin/bips/blob/e918b50731397872ad2922a1b08a5a4cd1d6d546/scripts/buildtable.pl#L98

What's the workflow of the bitcoin.it webmaster once this is merged? Would your merge this if a compiler is needed and missing and no workaround is suggested? See: https://en.bitcoin.it/wiki/Category:BIP

Concept ACK

**Comment by luke-jr** [comment]
BIP 1 supported markdown, but it was removed in BIP 2. I forget why - maybe simply lack of real-world usage?

Markdown does certainly seem more popular than mediawiki format these days.

Amending BIP 2 seems like an unclear proposition, however. First, someone needs to establish that there's some kind of consensus from participants, and secondly that amending Final BIPs isn't technically correct.

**Comment by katesalazar** [comment]
Hello, throwing a bunch of questions in the air

> BIP 1 supported markdown, but it was removed in BIP 2. I forget why - maybe simply lack of real-world usage?

Note (from Wikipedia) "The initial description of Markdown[[10]](https://en.wikipedia.org/wiki/Markdown#cite_note-10) contained ambiguities and raised unanswered questions, causing implementations to both intentionally and accidentally diverge from the original version. This was addressed in 2014 when long-standing Markdown contributors released [CommonMark](https://en.wikipedia.org/wiki/Markdown#Standardization), an unambiguous specification and test suite for Markdown.[[11]](https://en.wikipedia.org/wiki/Markdown#cite_note-FutureOfMarkdown-11)".

> Markdown does certainly seem more popular than mediawiki format these days.

Do you mean which flavor?

> Amending BIP 2 seems like an unclear proposition, however. First, someone needs to establish that there's some kind of consensus from participants, and secondly that amending Final BIPs isn't technically correct.

[Are BIP Comments abandoned?][0] Any other point in discussion about BIP 2? Does it make sense to replace BIP 2 with new process?

[0]: https://bitcoin.stackexchange.com/a/119375/126793

**Comment by kallewoof** [comment]
@ajtowns points out that the BIP isn't final, so amending it with a minor change like the suggested one seems reasonable.

Unless there is active opposition towards this, I propose this is merged.

**Comment by darosior** [comment]
Concept ACK.

**Comment by katesalazar** [comment]
wut
does this mean, first BIP written in Markdown has to ship the necessary changes in buildtable.pl?
can we do the things in the correct order

**Comment by murchandamus** [comment]
Does https://github.com/bitcoin/bips/pull/1577 address your concern, @katesalazar?

**Pull request merged**
